tests/custom_hypothesis_support.py

```diff
@@ -63,6 +71,13 @@ def heterogeneous_dictionary(**kwargs):
     return hs.dictionaries(index_values, primitive_values, **kwargs)
 
 
+def boolean_operator():
+    """Return a strategy which generates a boolean operator symbol (string)."""
+    return hs.one_of(hs.text('+-/*%', min_size=1, max_size=1),
```

Comment:
This is not an appropriate use of `hs.text`. You should use a strategy that takes as input a list of strings, and returns one of the strings.